src/Maestro/Maestro.ContainerApp/Actors/PullRequestActor.cs

```diff
@@ -54,15 +60,20 @@ public abstract class PullRequestActor : IPullRequestActor
         _logger = loggerFactory.CreateLogger(GetType());
         _redisCache = redis.GetDatabase();
         _actorId = actorId;
+        _reminders = reminders;
 
         PullRequestRedisKey = PullRequest + _actorId.Id;
         PullRequestUpdateRedisKey = PullRequest + _actorId.Id;
         PullRequestCheckRedisKey = PullRequestCheck + _actorId.Id;
+        PullRequestCheckReminder = PullRequestCheckReminderPrefix + _actorId.Id;
+        PullRequestUpdateReminder = PullRequestUpdateReminderPrefix + _actorId.Id;
     }
 
     protected string PullRequestRedisKey { get; }
     protected string PullRequestUpdateRedisKey { get; }
     protected string PullRequestCheckRedisKey { get; }
+    protected string PullRequestCheckReminder { get; }
+    protected string PullRequestUpdateReminder { get; }
```

Comment:
Why not just 
```csharp
protected string PullRequestCheckReminder => PullRequestCheckReminderPrefix + _actorId.Id;
```

Reply:
I made it the same as PullRequestUpdateRedisKey and the others

Reply:
Yeah, those are also not great. But what the hell, it's a hackathon :)